### PR TITLE
Go-back button added

### PR DESCRIPTION
**Pull request by JM066**

### CODE REVIEW

File: pages/styles/globals.scss [hunk 1]
```diff
@@ -97,7 +97,9 @@ a {
   text-decoration: none;
 }
 
-* {
+*,
+*::before,
```
Comment (ivanms1) [inline pages/styles/globals.scss:101]: why is this necessary?

File: src/components/Network/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  {
+    value: "testnet",
+    label: "Conun Testnet",
+  },
+  {
+    value: "mainnet",
+    label: "Conun Mainnet",
+  },
+];
+// const CLICK_FORWARDS_OPTIONS: Step[] = [
```
Comment (ivanms1) [inline src/components/Network/index.tsx:17]: remove please

File: src/components/Network/index.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import classNames from "classnames";
+import styles from "./Network.module.scss";
+
+// import { Step } from "@/store/store";
```
Comment (ivanms1) [inline src/components/Network/index.tsx:5]: remove please

File: src/pages/Intro/HelpUs/index.tsx [hunk 4]
```diff
@@ -28,14 +29,25 @@ function HelpUs() {
...
-        <GoBack className={styles.Goback} />
-        <div className={styles.Setting}>
-          <div className={styles.Network}>CONUN TESTNET</div>
-          <Setting />
+      <div className={styles.Header}>
+        <div className={styles.Nav}>
+          <Button
+            noStyle
+            onClick={() => {
+              console.log("current", currentStep.previous);
```
Comment (ivanms1) [inline src/pages/Intro/HelpUs/index.tsx:37]: remove

File: src/store/store.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { getIsLoggerActive, setIsLoggerActive } from "@/helpers/logger";
 import create from "zustand";
 
-type Step =
+export type Step =
```
Comment (ivanms1) [inline src/store/store.ts:4]: export is not needed anymore

File: pages/styles/globals.scss (hunk 1 above)
Comment (JM066) [reply pages/styles/globals.scss:101]: I just read that it includes all elements including pseudo elements so I added it.

File: src/pages/Intro/HelpUs/HelpUs.module.scss [hunk 6]
```diff
@@ -50,14 +31,31 @@
...
+    & > div {
+      display: flex;
+
+      &:first-child {
+        font-weight: 500;
+      }
+      & .RedBadge {
+        width: 15px;
+      }
+      & p {
```
Comment (ivanms1) [inline src/pages/Intro/HelpUs/HelpUs.module.scss:44]:
```suggestion
      & > p {
```
or add a classname

File: src/pages/Intro/HelpUs/HelpUs.module.scss [hunk 7]
```diff
@@ -50,14 +31,31 @@
...
+        letter-spacing: -0.03em;
+        margin-left: 5px;
+        max-width: 280px;
+        word-break: break-word;
+
+        & span {
```
Comment (ivanms1) [inline src/pages/Intro/HelpUs/HelpUs.module.scss:50]:
```suggestion
        & > span {
```

**Review by ivanms1 — APPROVED** [review]
just some small changes
